pricing-page-exercise/onder/styles.css

```diff
@@ -0,0 +1,215 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    font-family: "Open Sans", sans-serif;
+    font-size: 16px;
+}
+body {
+    margin: 0;
+    padding: 0;
+  background-color: rgb(220, 220, 220);
+}
+/* navbar section */
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1.25rem;
+    background: rgb(180, 180, 180);
+    height: 70px;
+    border-bottom: 1px solid rgb(222 226 230);
+    box-shadow: rgba(0, 0, 0, 0.05) 0px 7px 11px 0px;  
+}
+
+.heading-small {
+    font-size: 1.2rem;
+    font-weight: 400;
+    text-align: left;
+}
+
+.navbar ul {
+    display: flex;
+    align-items: center;
+    gap: 1.2rem; 
+    list-style-type: none;
+}
+
+.navlink {
+    text-decoration: none;
+    color: #363636;
+    font-size: 1rem;
+}
+.navlink:hover {
+    text-decoration:underline;
+    color:#363636;
+}
+
+.text-center {
+    text-align: center;
+  font-weight: bold;
+}
+
+
+.btn {
+    color:rgb(30, 143, 218);
+    border: 1px solid rgb(30, 143, 218);
+    border-radius: 5px;
+    padding: 0.6rem; 
+    text-decoration: none;   
+}
+.btn:hover {
+    background-color: rgb(30, 143, 218);
+    color:white;
+}
+.btn:active {
+    box-shadow: 0 0 2px 2px #3e8fc5;
+}
+
+
+/* description section */
+.description {
+    background: (255, 255, 255);
+    max-width: 700px;
+    width: 70%;
+    padding: 0.5rem;
+    margin-bottom: 4rem;
+    margin-top: 4rem;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
When you only need to set margin for some of the sides, using a shorthand property is common: `margin: 4rem auto;`